### PR TITLE
Added Japan-only Walda PC icons

### PR DESCRIPTION
**Pull request by fdeblasio**
## Description
The custom wallpapers obtained from telling Mr. Pepper a password are missing three icons that were only in the Japan release. This PR replaces their blank images with the actual icons

## Images
bolts

## **Discord contact info**
Frankfurter0

### CODE REVIEW

**Review by Bassoonian — CHANGES_REQUESTED** [review]
It may be worth putting your changes behind a config so that the user can choose which set of icons to use (you could have a single one for all icons, or have each icon have a separate config, or perhaps a combination of both).

**Comment by fdeblasio** [comment]
> It may be worth putting your changes behind a config so that the user can choose which set of icons to use (you could have a single one for all icons, or have each icon have a separate config, or perhaps a combination of both).

I feel like the name of the config is a little clunky, but it's in now

File: include/config/overworld.h [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 // PC settings
 #define OW_PC_PRESS_B               GEN_LATEST // In Gen4, pressing B when holding a Pokémon is equivalent to placing it. In Gen3, it gives the "You're holding a Pokémon!" error.
+#define OW_PC_JAPAN_WALDA_ICONS     FALSE // The Japanese release has the Cross, Bolt, and Plusle icons for Walda's wallpapers, but in the US release, they're blank
```
Comment (AsparagusEduardo) [inline include/config/overworld.h:12]: I think it'd be nice to have this on by default, as not many people even know of Walda's wallpapers in the first place, and even less would bother to change this on their own.
```suggestion
#define OW_PC_JAPAN_WALDA_ICONS     TRUE       // In the US release of Emerald, the Cross, Bolt, and Plusle icons for Walda's wallpapers were left blank from the Japan release. Setting this to TRUE will restore them.
```
Besides that and the proposed comment change, looks good to me :)

Comment (fdeblasio) [reply include/config/overworld.h]: Just commited the changes! Describing things has never been my forte so thanks for including a better description

**Review by Bassoonian — APPROVED** [review]

**Pull request merged**